### PR TITLE
Add octopus energy

### PR DESCRIPTION
**Pull request by kieronlanning**

### CODE REVIEW

**Comment by kieronlanning** [comment]
I'm not sure why this is failing with the following error:

```
Error:  'Format' error in entries/o/octopus.energy.json
- tag: url
  data: octopus.energy
  expected: uri
```

I did try adding `https://octopus.energy` which fails with a different error.

File: entries/o/octopus.energy.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "Octopus Energy": {
+    "domain": "octopus.energy",
+    "url": "octopus.energy",
```
Comment (phallobst) [inline entries/o/octopus.energy.json]: If the `uri` field is given, it must contain a valid URI (e.g. start with `https://`). However, if the value is `https://`+domain, it should be left out.
```suggestion
```

**Review by phallobst — CHANGES_REQUESTED** [review]

**Comment by phallobst** [comment]
Here's an SVG version of the logo, which we prefer.
octopus energy

**Comment by kieronlanning** [comment]
@phallobst didn't see you svg comment - sorted now, I did look but couldn't find one initially.

File: entries/o/octopus.energy.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "Octopus Energy": {
+    "domain": "octopus.energy",
```
Comment (phallobst) [inline entries/o/octopus.energy.json:3]:
```suggestion
    "domain": "octopus.energy",
    "additional-domains": [
      "octopusenergy.de",
      "octopusenergy.es",
      "octopusenergy.fr",
      "octopusenergy.it",
      "octopusenergy.co.jp",
      "octopusenergy.nz",
      "octopusenergy.com"
    ],
```

File: entries/o/octopus.energy.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    "contact": {
+      "facebook": "octopusenergy",
+      "twitter": "OctopusEnergy"
+    },
+    "categories": [
+      "utilities"
+    ]
```
Comment (phallobst) [inline entries/o/octopus.energy.json:29]:
```suggestion
    ],
    "regions": [
      "gb",
      "de",
      "es",
      "fr",
      "it",
      "jp",
      "nz",
      "us"
    ]
```

**Review by phallobst — CHANGES_REQUESTED** [review]

**Review by phallobst — APPROVED** [review]

**Pull request merged**